### PR TITLE
User api documents

### PR DESCRIPTION
**Pull request by dungvan**

### CODE REVIEW

File: docs/api-references/user.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                "full_name": "dungvan2512",
+                "email": "dungvan@test.email.com",
+                "birthday": null,
+                "gender": 0,
+                "address": null,
+                "phone_number": null,
+                "image": "http://foodmarket.com/images/users/default.jpg",
+                "is_admin": 1,
+                "is_active": 0,
+                "created_at": "2017-08-23 04:13:40",
```
Comment (ghost) [inline docs/api-references/user.md]: If this API don't need to use `created_at `, `updated_at`, `deleted_at`. You don't need to return them. Select only value feature to want to use.

File: docs/api-references/user.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+## User API
+
+### `GET` User
+```
+/api/user/
```
Comment (ghost) [inline docs/api-references/user.md]: Resource `user` should be `users`. You can check on: https://laravel.com/docs/5.4/controllers#resource-controllers

File: docs/api-references/user.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
+```
+#### Sample Response
+```json
+{
+    "data": {
+                "id": 74,
```
Comment (ghost) [inline docs/api-references/user.md]: Indent 1 level for child data.

File: docs/api-references/user.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+            },
+    "success": true
+}
+```
+
+### `Post` user
+```
+/api/user/create
+```
+Registry a food
```
Comment (ghost) [inline docs/api-references/user.md]: Please fill correct API description.

File: docs/api-references/user.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                "updated_at": "2017-08-23 04:13:40",
+                "deleted_at": null
+            },
+    "success": true
+}
+```
+
+### `Post` user
+```
+/api/user/create
```
Comment (ghost) [inline docs/api-references/user.md]: Please change endpoint to use `users` resource.

File: docs/api-references/user.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+|---|---|---|---|
+| full_name | String | required | Name of user |
+| email | Email | required | email to login |
+| password | Password | required, least at 6 character | password |
+| password_confirmation | Password | required, map to password | password confirmation |
+
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
```
Comment (ghost) [inline docs/api-references/user.md]: I think this API doesn't need Authorization

File: docs/api-references/user.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+   "password_confirmation": "123456"
+}
+```
+
+#### Sample Response
+```json
+{
+    "data": {
+        "id": 74,
+                        "full_name": "Van Duc Dung",
```
Comment (ghost) [inline docs/api-references/user.md]: Indent 1 level for child data.

File: docs/api-references/user.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+                        "created_at": "2017-08-23 04:13:40",
+                        "updated_at": "2017-08-23 04:13:40",
+                        "deleted_at": null
+    },
+    "success": true
+}
+```
+### `Put` user
+```
+/api/user/edit
```
Comment (ghost) [inline docs/api-references/user.md]: Please change endpoint to use `users` resource.

File: docs/api-references/user.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+| gender | Integer | required | Default 0 : female, 1 : male  |
+| address | String | required | address of user |
+| phone_number | String | required, number | phone number of user |
+| password | Password | required, least at 6 character | password |
+| password_confirmation | Password | nullable, map to password | password confirmation |
+
+#### Request header
+```
+  "Content-Type": "application/json",
+  "Accept": "application/json",
```
Comment (ghost) [inline docs/api-references/user.md]: `"Accept": "application/json",` is OK, but why only this API has `Accept`. Please make unification for all APIs.

File: docs/api-references/user.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+  "Authorization": "{token_type} {access_token}"
+```
+
+#### Sample Request body
+```json
+{
+"full_name":"dungvan25123",
```
Comment (ghost) [inline docs/api-references/user.md]: Please use indent for JSON structure.

File: docs/api-references/user.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+"birthday":"1994-03-12",
+"gender":1,
+"address":"DienDuong"
+,"phone_number":"123456789"
```
Comment (ghost) [inline docs/api-references/user.md]: Please use correct comma symbol.

**Review by ghost — COMMENTED** [review]
Please check my comment.

File: docs/api-references/user.md (hunk 6 above)
Comment (dungvan) [reply docs/api-references/user.md]: this API get profile of current user login in client, so we need to Authorization to get data

File: docs/api-references/user.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
+## User API
+
+### `GET` User
+```
+/api/users/
```
Comment (ghost) [inline docs/api-references/user.md]: To display information of current user login, you can use endpoint `/api/users/me/`

File: docs/api-references/user.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+    "is_admin": 1,
+    "is_active": 0
+  },
+  "success": true
+}
+```
+
+### `Post` user
+```
+/api/users/create
```
Comment (ghost) [inline docs/api-references/user.md]: To create a user. user endpoint `/api/users/` with method `POST`.
You can check in this document: https://laravel.com/docs/5.4/controllers#resource-controllers

File: docs/api-references/user.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+      "is_admin": 1,
+      "is_active": 0
+    },
+  "success": true,
+  "message": "Registry successfully!, please login and enjoy!"
+}
+```
+### `Put` user
+```
+/api/users/edit
```
Comment (ghost) [inline docs/api-references/user.md]: To update current user. Use endpoint `/api/users/me/` with method `PUT`

**Review by ghost — COMMENTED** [review]
Please check my comment.

File: docs/api-references/user.md [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+```
+/api/users
+```
+Registry a user
+
+#### Parameters
+| Key | Type | Required | Description |
+|---|---|---|---|
+| full_name | String | required | Name of user |
+| email | Email | required | email to login |
```
Comment (ghost) [inline docs/api-references/user.md]: In JSON, don't have type **Email**, please use type of data String, Integer, Float, Boolean

File: docs/api-references/user.md [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+| password | Password | required, least at 6 character | password |
```
Comment (ghost) [inline docs/api-references/user.md]: **Required** is not validation rule. Just define **Required** or **Optional**
If you want to clear this key, add more info on Description column.

**Review by ghost — COMMENTED** [review]
Please check my comment

File: docs/api-references/user.md [hunk 17]
```diff
@@ -0,0 +1,125 @@
...
+```
+/api/users/me
+```
+Update profile of current user
+
+#### Parameters
+| Key | Type | Required | Description |
+|---|---|---|---|
+| full_name | String | required | Name of user |
+| birthday | Date | required | email to login |
```
Comment (ghost) [inline docs/api-references/user.md]: JSON doesn't have Date type, please use String.

**Review by ghost — COMMENTED** [review]
Please check my comment.

**Comment by ghost** [comment]
LGTM 👍

**Review by ghost — APPROVED** [review]

**Pull request merged**